### ISSUE TITLE
View Plan & View Census Button Colors

### ISSUE THREAD

**Issue by birdsong** [CityOfPhiladelphia/slash-data#8]
Please make these buttons the same color as the light blue links (like the Right To Know Request Form), turning them a darker shade when hoovered over.

**Comment by majshehab** [comment]
mockup has the background of the boxes in a darker color, which could help the white buttons pop more, as well. what do you think, @birdsong ?
![screen shot 2014-10-27 at 10 33 00 am](https://cloud.githubusercontent.com/assets/4324930/4792373/d8d28c44-5de5-11e4-8f24-0e56bb02b583.png)

**Comment by birdsong** [comment]
Yes, I agree that having the top two boxes with a darker background helps differentiate them from the lower row of boxes as well. @gsf

**Comment by majshehab** [comment]
@gsf no divider 
background-color: #CECECE;
border: 1px solid #bcbbbb;

button: #fff text #333 bg color

(and keep the 3 boxes in "Data" white/ as is)

**Comment by gsf** [comment]
See the panels now. @majshehab, we had talked about throwing a little of that bluish color in there on the buttons. Thoughts?

Also (and maybe related to the blue), do we want any change on hover?
